list1.py

```diff
@@ -37,8 +36,13 @@ def match_ends(words):
 # Hint: this can be done by making 2 lists and sorting each of them
 # before combining them.
 def front_x(words):
-    """Your code goes here.  Edit this docstring."""
-    return
+    # not golf answer
+    # x_sorted = sorted([word for word in words if (word[0] == "x")])
+    # not_x_sorted = sorted([word for word in words if (word[0] != "x")])
+    # return(x_sorted + not_x_sorted)
+
+    # golf answer
```

Comment:
Technically possible, I would suggest breaking each piece of data onto it's own line.
Codewrds however, would be proud.